Makefile

```diff
@@ -8,6 +8,7 @@ mypy:
 	mypy --ignore-missing-imports $(MODULES)
 
 test: lint mypy
+	tests/fixtures/populate.py --s3-bucket $(DSS_S3_BUCKET_TEST_FIXTURES) --gs-bucket $(DSS_GS_BUCKET_TEST_FIXTURES)
```

Comment:
I agree that generally that we should be using randomly generated UUIDs.  However, I ran into the problem when I was building the initial producer API tests.

PUT is easy to verify.  Write something and then read it back.  But how does one verify GET without knowing PUT works?  The only way I could think of was to write an authoritative thing by hand and GET'ing it.  I don't think making tests for PUT relying on a known functioning GET and tests for GET relying on a known functioning PUT is particularly good.

Happy to entertain other suggestions if you have any. :)